### ISSUE TITLE
Issue with the current commit

### ISSUE THREAD

**Issue by hardik22317** [AlexeyAB/darknet#7439]
Hello @AlexeyAB,

I guess there is some issue with the current commit. I am just cloning darknet, after that setting GPU, CUDNN, CUDNN_HALF, and OPENCV to 1. After that, while running the make command, I am getting following error. 
![ss-alexey](https://user-images.githubusercontent.com/22682878/109422697-a9aad300-7a02-11eb-86cf-f29011915d92.png)

Please have a look at it and kindly make the required changes.

**Comment by rebotnix** [comment]
We have same @hardik22317 ,

/src/network_kernels.cu(706): error: identifier "cudaStreamCaptureModeGlobal" is undefined

./src/network_kernels.cu(706): error: too many arguments in function call

2 errors detected in the compilation of "/tmp/tmpxft_00007159_00000000-7_network_kernels.compute_70.cpp1.ii".
Makefile:185: recipe for target 'obj/network_kernels.o' failed
make: * [obj/network_kernels.o] Error 1

Any idea to fix?

**Comment by takashide** [comment]
I met same result.
Probably It was caused by old cuda.

https://forums.developer.nvidia.com/t/cuda-10-1-simplecudagraphs-doesnt-compile/71283

My cuda version is 10.0. 
I'm still investigating.

Looking for someone's ideas!
Thank you

**Comment by rebotnix** [comment]
Same with CUDA 10.1 and 10.2.

**Comment by kaiser24** [comment]
I'm facing the same issue with CUDA 10.0 and GPU=1 and CUDNN=1

**Comment by gudianyue** [comment]
Same with CUDA 10.0 and GPU=1 and CUDNN=1 and OPENCV=1.

./src/network_kernels.cu(364): warning: variable "l" was declared but never referenced

./src/network_kernels.cu(706): error: identifier "cudaStreamCaptureModeGlobal" is undefined

./src/network_kernels.cu(706): error: too many arguments in function call

2 errors detected in the compilation of "/tmp/tmpxft_000057a0_00000000-6_network_kernels.cpp1.ii".
Makefile:187: recipe for target 'obj/network_kernels.o' failed
make: *** [obj/network_kernels.o] Error 1

Please tell me how to fix it.

**Comment by xd905** [comment]
> I'm facing the same issue with CUDA 10.0 and GPU=1 and CUDNN=1

Have you solved it?

**Comment by kaiser24** [comment]
> > I'm facing the same issue with CUDA 10.0 and GPU=1 and CUDNN=1
> 
> Have you solved it?

not exactly, I ended up going back some commits and it worked but I guess is still there with the current commit

**Comment by xd905** [comment]
> > > I'm facing the same issue with CUDA 10.0 and GPU=1 and CUDNN=1
> > 
> > 
> > Have you solved it?
> 
> not exactly, I ended up going back some commits and it worked but I guess the problem is still there with the current commit

Have you changed the version of CUDA？

**Comment by stephanecharette** [comment]
Note this is a new limitation.  See here:  https://github.com/AlexeyAB/darknet/issues/7486#issuecomment-794056697
(See issue #7486, issue #7439, and issue #7452.)

**Comment by xd905** [comment]
> Note this is a new limitation. See here: [#7486 (comment)](https://github.com/AlexeyAB/darknet/issues/7486#issuecomment-794056697)
> (See issue #7486, issue #7439, and issue #7452.)

I can see that. Thank you very much

**Comment by marcusturewicz** [comment]
I updated https://github.com/AlexeyAB/darknet/blob/22cb99e0418da26dc0d4d0587f99dc113cd2f559/src/network_kernels.cu#L706 to 
```
CHECK_CUDA(cudaStreamBeginCapture(stream0));
```

and it compiled for me.

**Comment by AlexeyAB** [comment]
> Same with CUDA 10.1 and 10.2.

> I'm facing the same issue with CUDA 10.0 and GPU=1 and CUDNN=1

It seems that `cudaStreamCaptureModeGlobal` was added in CUDA 10.1 https://docs.nvidia.com/cuda/archive/10.1/cuda-runtime-api/group__CUDART__STREAM.html

So it should work in CUDA 10.1, 10.2, 11.0, ...

![image](https://user-images.githubusercontent.com/4096485/112246140-3d1e9f00-8c63-11eb-93c6-cd5cf3025fb4.png)


----

While I can't find it in CUDA 10.0: https://docs.nvidia.com/cuda/archive/10.0/cuda-runtime-api/group__CUDART__STREAM.html

**Comment by corentin87** [comment]
It did not work for me with CUDA 11.0. I had to do the following as well:
`CHECK_CUDA(cudaStreamBeginCapture(stream0, cudaStreamCaptureModeGlobal));` to `CHECK_CUDA(cudaStreamBeginCapture(stream0));`

and it compiled fine.
